web/themes/custom/lex/gulpfile.js

```diff
@@ -59,17 +59,49 @@ return gulp.src('./scss/**/*.scss')
 //     .on('close', done);
 // });
 
+/**
+ * @task refresh
+ * Refresh browser
+ */
+gulp.task('refresh', function(done) {
+    browserSync.reload();
```

Comment:
This is a very small part of the browsersync solution. You're missing a call to `browserSync.init` that starts listening on a port. Then there's a custom module that includes the code on every page to _**connect**_ to the the browsersync run by gulp.

Search the ukhc codebase for `ukhc-sync.lndo.site` if you want to fully implement it. Otherwise, this can go.

Reply:
Looks like you're still missing the browsersync drupal module which will connect to the development server